packages/cms/store/resource.ts

```diff
@@ -64,6 +64,11 @@ interface ResourceState extends State {
 
 interface ResourceMethods extends State {
   findResource: (slug: string) => ResourceContract
+  fetchTableData: (
+    slug: string,
+    page?: number,
+    perPage?: number
+  ) => Promise<any>
```

Comment:
The return type can be more unique. Please check the `store/auth.ts` for an example.